src/app/login/login.component.ts

```diff
@@ -7,8 +9,32 @@ import { Component } from '@angular/core';
 })
 export class LoginComponent {
   
-  constructor()
+  constructor(private loginService: LoginService,private router: Router)
   {
     console.log("hey2")
   }
+
+  login: Login = 
+  {
+    login: null,
+    password: null,
+    role: null
+  }
+
+  blad: string | undefined;
```

Comment:
nie polskie + jak uzyjesz reactive to nie bedziesz musiała przechowywac bledu w zmiennej